build.gradle

```diff
@@ -119,7 +119,8 @@ dependencies {
 
     // ZipArchive dependencies used for integration tests
     zipArchive group: 'org.opensearch.plugin', name:'opensearch-ml-plugin', version: "${version}"
-    zipArchive group: 'org.opensearch.plugin', name:'opensearch-job-scheduler', version: "${version}"
+    //JS plugin is published to `org/opensearch` instead of `org/opensearch/plugin` under local maven repo: https://mvnrepository.com/artifact/org.opensearch/opensearch-job-scheduler.
+    zipArchive group: 'org.opensearch', name:'opensearch-job-scheduler', version: "${version}"
```

Comment:
we also have https://mvnrepository.com/artifact/org.opensearch.plugin/opensearch-job-scheduler

And the groupId is `org.opensearch.plugin` defined here https://github.com/opensearch-project/job-scheduler/blob/main/build.gradle#L131

Reply:
Job-scheduler is not publishing plugin zip in `publishToMavenLocal` task so can't find plugin in local repo, changing this makes it points to jar file directly, but I checked again and found we'd better directly depend on jars instead of zip for both job-scheduler and anomaly-detection. I've created this PR to fix this: https://github.com/opensearch-project/skills/pull/74. And this one can be closed.